src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -71,6 +71,8 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             // shouldn't reach here
             _ => throw new ArgumentException($"Unknown browser : {Arguments.Browser}")
         };
+        // Browser respects LANGUAGE in the first place, only if empty it checks LANG
+        Environment.SetEnvironmentVariable("BROWSER_LANGUAGE", Arguments.Locale);
```

Comment:
There is `--lang` for chrome. Is that better ?

Reply:
It does not work on Linux, https://developer.chrome.com/docs/extensions/reference/i18n/#linux

Reply:
We have been using `--lang` with debugger tests - https://github.com/dotnet/runtime/blob/main/src/mono/wasm/debugger/DebuggerTestSuite/ChromeProvider.cs#L166-L167 . Could you also confirm that it doesn't actually work there, and update if needed?

Reply:
So far it works because we are loading full ICU in debugger tests. As soon as we merge ICU sharding PR the line you quoted will be good for Windows but will fail with Linux. I already fixed it on that PR, it's in this commit: https://github.com/dotnet/runtime/pull/80421/commits/dde05ee00bf0aecacfbfe6e302f34f65e272e2d2.